discovery/chan_series.go

```diff
@@ -136,7 +136,21 @@ func (c *ChanSeries) UpdatesInHorizon(chain chainhash.Hash,
 			return nil, err
 		}
 
-		updates = append(updates, chanAnn)
+		// Create a slice to hold the `channel_announcement` and
```

Comment:
should we expand one of the ApplyGossipFilter unit tests to cover this?

Reply:
+1

Reply:
This method is mocked in the `ApplyGossipFilter` tests, I also find it weird to test it there. I will build the tests in a separate PR given it's a bit involved.